src/routes/boot.ts

```diff
@@ -476,6 +476,10 @@ const loggedInBoot = async ({
     const isTeamMember = exp?.a?.team === 1;
     const isPlus = isPlusMember(user.subscriptionFlags?.cycle);
 
+    if (isPlus) {
+      exp.a.plus = 1;
```

Comment:
Is `a` object here, if its undefined it will break?

Reply:
Defaults to empty object I think:
https://github.com/dailydotdev/daily-api/blob/cfd63715cf57b59672611a93e514172711883153/src/routes/boot.ts#L362-L365

Reply:
jep!